### ISSUE TITLE
how to save and load model?

### ISSUE THREAD

**Issue by wslhhh1994** [ar4/deepwave#24]
when use deepwave ?how save the trained model?And how to load it ?

**Comment by ar4** [comment]
Hello again,

The model is a regular PyTorch Tensor. For example, in the [FWI example](https://colab.research.google.com/drive/1PMO1rFAaibRjwjhBuyH3dLQ1sW5wfec_?usp=sharing), the model is the PyTorch Tensor `model`. You can save and load it like a regular PyTorch Tensor. If you prefer, you can also convert it to a NumPy Array: model.detach().cpu().numpy() and then you can use normal NumPy methods for saving and loading.
